src/Marketing/InstalledExtensions.php

```diff
@@ -275,7 +275,7 @@ protected static function get_amazon_ebay_extension_data() {
 			}
 
 			$data['settingsUrl'] = admin_url( 'admin.php?page=codisto-settings' );
-			$data['docsUrl']     = 'https://docs.woocommerce.com/document/getting-started-with-woocommerce-amazon-ebay-integration/?utm_medium=product';
+			$data['docsUrl']     = 'https://woocommerce.com/documentation/getting-started-with-woocommerce-amazon-ebay-integration/?utm_medium=product';
```

Comment:
Though it's not in the allowed plugin list, this plugin seems no longer available and the document link is 404.

I think this is the new plugin by Codisto.
https://woocommerce.com/products/amazon-ebay-integration/

Reply:
If we're longer need this. I think we can just remove it. Please help confirm this, thanks.

Reply:
@chihsuan I think you're correct on it. I installed `https://woocommerce.com/products/amazon-ebay-integration/` and confirm the plugin has the same setting page path (page=codisto-settings).

Reply:
@chihsuan Actually I've found the new documentation page in https://woocommerce.com/document/multichannel-for-woocommerce-google-amazon-ebay-walmart-integration/ from the product page. I think we can replace it with this instead

Reply:
Updated. Thanks for confirming! 😄